packages/shared/src/utils.ts

```diff
@@ -203,3 +203,63 @@ export const fileSizeToBytes = (input: string) => {
   const bytes = sizeValue * Math.pow(1024, FILESIZE_UNITS.indexOf(sizeUnit));
   return Math.floor(bytes);
 };
+
+// This is absolutely horrendous, but we need some way to resole environment
+// variables regardless of build tooling. If you have a better idea, please
+// fix this.
+export const DANGEROUS__checkEnvironmentVariable = (variable: string) => {
```

Comment:
this doesn't work as the variable names has to be statically analyzable

Reply:
<img width="824" alt="image" src="https://github.com/pingdotgg/uploadthing/assets/55754540/7b5dc20b-107a-4f74-b084-8cafa4450add">
It seems that we already have an idea of a solution for that

Reply:
Screenshot from https://github.com/pingdotgg/uploadthing/pull/292

Reply:
> this doesn't work as the variable names has to be statically analyzable

Yup 😅 planning to switch to the alternative buildEnv() method you proposed in #292, though it will need to be a bit more complicated to handle the various potential prefixes.